crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -550,6 +550,29 @@ impl<'w, 's> Commands<'w, 's> {
             .push(RunSystemWithInput::new_with_input(id, input));
     }
 
+    /// Registers a system and returns a [`SystemId`] so it can later be called by [`World::run_system`].
+    ///
+    /// It's possible to register the same systems more than once, they'll be stored separately.
```

Comment:
This is a really important consideration: we should try to call this out extremely carefully.

IMO, a doc test here showing how to check if a system is already registered would be good, or perhaps a secondary API that is idempotent and does nothing if an existing registration is found.

Reply:
Is there an existing way to verify if the System is already inserted? The regular `World::register_system` have the same possible problem

Reply:
Hmm, I don't think so 🤔 @Trashtalk217 may know though. This is a particularly nasty footgun with commands though, since it's easy to stick them into a regular system and register an unbounded number of them.